docs/tutorials/gymnasium_basics/load_quadruped_model.md

```diff
@@ -0,0 +1,230 @@
+Load custom quadruped robot environments
+================================
+
+In this tutorial, we will see how to use the `MuJoCo/Ant-v5` framework to create a quadruped walking environment. Using a model file (ending in `.xml`) without having to create a new class.
+
+Steps:
+
+0. Get your **MJCF** (or **URDF**) model file of your robot.
+	1. Create your own model (see the [Guide](https://mujoco.readthedocs.io/en/stable/modeling.html)).
+	2. Find a ready-made model (in this tutorial, we will use a model from the [**MuJoCo Menagerie**](https://github.com/google-deepmind/mujoco_menagerie) collection).
+1. Load the model with the `xml_file` argument.
+2. Tweak the environment parameters to get the desired behavior.
+ 	1. Tweak the environment simulation parameters.
+	2. Tweak the environment termination parameters.
+	3. Tweak the environment reward parameters.
+	4. Tweak the environment observation parameters.
+3. Train an agent to walk/run with your robot.
+
+
+The reader is expected to be familiar with the basics of robotics and the included `Gymnasium/MuJoCo` environments and the robot model they use, familiarity with the **MJCF** file model format and the `MuJoCo` simulator is not required but is recommended.
+
+Setup
+------
+We will need `gymnasium>=1.0.0`.
+
+```sh
+pip install gymnasium>=1.0.0
+```
+Step 0.2 our robot model
```

Comment:
Step 0.2 - Creating a Robot Model

I would repeat this style for all of the other titles